content/verification/veteran-verification/release-notes/2024-04-17.md

```diff
@@ -0,0 +1,4 @@
+We added a new endpoint: ACA Coverage (restricted access) for v2 of the Veteran Service History and Eligibility API.
+* Authorized users can send requests to this endpoint to check Veteran enrollment in minimum essential coverage (MEC) as defined by the Affordable Care Act (ACA). This enrollment inquiry is for a specified date range in requests (required).
```

Comment:
Does the required mean that a user has to submit a specified date range?

Reply:
Yes, it is required that the user submit a specific date range